L1Trigger/Phase2L1GMT/test/runGMT.py

```diff
@@ -144,7 +144,7 @@
 process.FEVTDEBUGHLToutput_step = cms.EndPath(process.FEVTDEBUGHLToutput)
 
 # Schedule definition
-process.schedule = cms.Schedule(process.raw2digi_step,process.L1TrackTrigger_step,process.testpath,process.endjob_step,process.FEVTDEBUGHLToutput_step)
+process.schedule = cms.Schedule(process.raw2digi_step,process.L1TTrackTrigger_step,process.testpath,process.endjob_step,process.FEVTDEBUGHLToutput_step)
```

Comment:
Isn't this still `L1TrackTrigger_step` ?

Reply:
Fixed